deps/base64/base64.gyp

```diff
@@ -78,6 +78,7 @@
       'include_dirs': [ 'base64/include', 'base64/lib' ],
       'sources': [ 'base64/lib/arch/ssse3/codec.c' ],
       'defines': [ 'BASE64_STATIC_DEFINE', 'HAVE_SSSE3=1' ],
+      'cflags_cpp': [],
```

Comment:
Do these work? What are the fixed warnings?

Reply:
Without these, you are passing-std=c++17 to clang-cl when compiling C code. The better solution would be for the build engine to recognize that it is compiling C code, and not pass the C++ flags.

Reply:
I'm surprised that it works because I don't see any occurence of `cflags_cpp` in the code base (including the source code of GYP)